### PR TITLE
ICP-2082: Set initial state of Z-Wave Switch Generic

### PR DESCRIPTION
**Pull request by munds**

### CODE REVIEW

File: devicetypes/smartthings/zwave-switch-generic.src/zwave-switch-generic.groovy [hunk 1]
```diff
@@ -64,11 +64,13 @@ metadata {
 def installed(){
 // Device-Watch simply pings if no device events received for checkInterval duration of 32min = 2 * 15min + 2min lag time
 	sendEvent(name: "checkInterval", value: 2 * 15 * 60 + 2 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"])
+	response(refresh())
 }
 
 def updated(){
 // Device-Watch simply pings if no device events received for checkInterval duration of 32min = 2 * 15min + 2min lag time
 	sendEvent(name: "checkInterval", value: 2 * 15 * 60 + 2 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"])
+	response(refresh())
```
Comment (mckeed) [inline devicetypes/smartthings/zwave-switch-generic.src/zwave-switch-generic.groovy]: I think it's only needed in installed(), not updated() too

**Comment by munds** [comment]
@mckeed I've made the changes you requested 👍 

Can you please do a code review whenever you get a chance!

cc: @workingmonk

**Comment by munds** [comment]
Whoa hold on, this doesn't look right. Let me redo the PR. :)

**Comment by munds** [comment]
New PR is https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/2522

**Pull request closed without merging**